### ISSUE TITLE
multiple identical notification

### ISSUE THREAD

**Issue by zitzman** [nisabek/icinga2-slack-notifications#4]
Not Sure why, but I get 3-4 alers per event in the Slack channel.
debug.log actually shows it run several times:

`[2017-03-27 13:04:50 +0000] debug/slack-notifications: Generated notification command successfully: curl --fail --connect-timeout 30 --max-time 60 --silent --show-error -X POST --data-urlencode 'payload={"channel":"#alerts","text":":red_circle: PROBLEM: Host <https://monitor.mydomain.com/icingaweb2/monitoring/host/show?host=RDGW-2|RDGW-2> is still in state DOWN. Plugin output: ```PING CRITICAL - Packet loss = 100%```","username":"icinga2"}' 'https://hooks.slack.com/services/xxxxxx/yyyyyy/zzzzzzz'
[2017-03-27 13:04:50 +0000] debug/slack-notifications: Generated notification command successfully: curl --fail --connect-timeout 30 --max-time 60 --silent --show-error -X POST --data-urlencode 'payload={"channel":"#alerts","text":":red_circle: PROBLEM: Host <https://monitor.mydomain.com/icingaweb2/monitoring/host/show?host=RDGW-2|RDGW-2> is still in state DOWN. Plugin output: ```PING CRITICAL - Packet loss = 100%```","username":"icinga2"}' 'https://hooks.slack.com/services/xxxxxx/yyyyyy/zzzzzzz'
[2017-03-27 13:04:50 +0000] debug/slack-notifications: Generated notification command successfully: curl --fail --connect-timeout 30 --max-time 60 --silent --show-error -X POST --data-urlencode 'payload={"channel":"#alerts","text":":red_circle: PROBLEM: Host <https://monitor.mydomain.com/icingaweb2/monitoring/host/show?host=RDGW-2|RDGW-2> is still in state DOWN. Plugin output: ```PING CRITICAL - Packet loss = 100%```","username":"icinga2"}' 'https://hooks.slack.com/services/xxxxxx/yyyyyy/zzzzzzz'
[2017-03-27 13:04:50 +0000] debug/slack-notifications: Generated notification command successfully: curl --fail --connect-timeout 30 --max-time 60 --silent --show-error -X POST --data-urlencode 'payload={"channel":"#alerts","text":":red_circle: PROBLEM: Host <https://monitor.mydomain.com`

**Comment by staenker** [comment]
Hi zitzman,
can you share the relevant config?
Cheers,
Richard

**Comment by mlp-ableton** [comment]
@zitzman I ran into the same issue just now. It seems the notifier [will send out one notification for each member in the `user_groups` variable](https://docs.icinga.com/icinga2/latest/doc/module/icinga2/chapter/monitoring-basics#alert-notifications), which defaults to `["icingaadmins"]` in `slack-nitifications-configuration.conf`.  Since I and several of my colleagues were members of this group, one notification for each member was sent to Slack.

I worked around this by creating a new `UserGroup` `slack`, which contains a single user `slack`.
